TRANSFORMERS/MATRIX_MANIPULATION/SHUFFLE_MATRIX/SHUFFLE_MATRIX.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+from flojoy import flojoy, Matrix
+
+
+@flojoy
+def SHUFFLE_MATRIX(
+    default: Matrix,
+) -> Matrix:
+    """The SHUFFLE_MATRIX node returns a matrix that is randomly shuffled by the first axis
```

Comment:
I think this should have a parameter allowing you to choose which axis would be shuffled